modules/src/script/setupModules.groovy

```diff
@@ -24,6 +28,7 @@ println "[INFO] Using HV version " + hvArtifactName;
 processFileInplace( hvModuleXml ) { text ->
     text.replaceAll( /hibernate-validator.*jar/, hvArtifactName )
 }
+appendDependency( hvModuleXml, "javax.money.api", true )
```

Comment:
Ah, of course. Not sure why, but somehow I thought that'd be the setup code within the integration test module.